src/3d/qgs3dmapscene.cpp

```diff
@@ -1095,6 +1095,16 @@ QgsDoubleRange Qgs3DMapScene::elevationRange() const
       yMin = std::min( yMin, zRange.lower() );
       yMax = std::max( yMax, zRange.upper() );
     }
+    else if ( layer->type() == Qgis::LayerType::Mesh )
+    {
+      QgsMeshLayer *meshLayer = qobject_cast< QgsMeshLayer *>( layer );
+      QgsMeshLayer3DRenderer *meshLayerRenderer = static_cast<QgsMeshLayer3DRenderer *>( meshLayer->renderer3D() );
```

Comment:
Can there be cases where `renderer3D()` returns `nullptr`?

Reply:
Good point. Checked now.